/**:
  ros__parameters:
    METERS_PER_INCH: 0.0254
    HOMOGRAPHIES:
      rss2025_14:
        PTS_IMAGE_PLANE: "[[204.0, 256.0], [457.0, 256.0], [148.0, 219.0], [490.0, 216.0]]"
        PTS_GROUND_PLANE: "[[11.0, 8.5], [11.0, -8.5],[22.0, 17.0],[22.0, -17.0]]"
      rss2025_8_old:
        PTS_IMAGE_PLANE: "[[429, 202],[254, 202],[424, 256],[54, 226]]"
        PTS_GROUND_PLANE: "[[56, -18.5],[56,14],[23,-7],[31,34.5]]"
        METERS_PER_INCH: 0.0254
      rss2025_8_new:
        PTS_IMAGE_PLANE: "None"
        PTS_GROUND_PLANE: "None"
